scripts/zones/Halvung/mobs/Copper_Borer.lua

```diff
@@ -4,6 +4,20 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 10, 0, 0)
```

Comment:
don't simply add the status effect unless its dispelable. also the video you linked shows 30 dmg not 10.

for perma spikes there is normally a function involved but if the damage never varies you may not need it. look at Sargas in attohwa chasm for an example. That script also shows how to set flags to prevent dispelling (since scripted spikes unfortunately still needed a status at the time it was written).

Reply:
moved this to AUTO_SPIKES as per your guidance